src/Catty/ViewController/Download/Details/ProjectDetailStoreViewController.m

```diff
@@ -160,21 +160,13 @@ -(void)downloadAgain:(id)sender
     [self downloadWithName:duplicateName];
 }
 
-#pragma mark - TTTAttributedLabelDelegate
-- (void)attributedLabel:(TTTAttributedLabel *)label didSelectLinkWithURL:(NSURL *)url
+- (void)reloadWithProject:(CatrobatProject *)loadedProject
```

Comment:
Could it be, that this method was added by mistake, or do you really need it in your PR?
Since the method was removed in the develop branch a few days ago.